sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -1416,4 +1416,21 @@ class StatisticsSuite extends StatisticsCollectionTestBase with TestHiveSingleto
       assert(catalogStats.rowCount.isEmpty)
     }
   }
+
+  test(s"CTAS should update statistics if ${SQLConf.AUTO_SIZE_UPDATE_ENABLED.key} is enabled") {
+    val tableName = "SPARK_23263"
+    Seq(false, true).foreach { updateEnabled =>
+      withSQLConf(SQLConf.AUTO_SIZE_UPDATE_ENABLED.key -> updateEnabled.toString) {
+        withTable(tableName) {
+          sql(s"CREATE TABLE $tableName STORED AS parquet AS SELECT 'a', 'b'")
```

Comment:
This looks affected by `spark.sql.hive.convertMetastoreParquet` automatically. Can we have a test coverage for both cases, `true` and `false`?

Reply:
Do we need change this test case because we have added test for data source CTAS command:
https://github.com/apache/spark/blob/974b8792205b2882f1b6c6f429c889dbf33434e6/sql/core/src/test/scala/org/apache/spark/sql/StatisticsCollectionSuite.scala#L600-L616

Reply:
Let's just add it here for now and dedup later.

Reply:
OK.